CommandLineToolExample/CommandLineToolExample.csproj

```diff
@@ -1,12 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <TargetFramework>netcoreapp6.0</TargetFramework>
-        <RootNamespace>YafcCommandLineToolExample</RootNamespace>
-        <LangVersion>8</LangVersion>
```

Comment:
Was this formatting automatic?

Reply:
Yes, unintentionally but this is what auto-format does for me. 2 spaces per xml property. I intentionally tried not to touch this formatting as much as possible, while still making it consistent among the files themselves. Looks like i failed here since the item group below does have 4 spaces indent.

Do we have a preference here? If so i update the .editorconfig and format all .csproj files that way.

Reply:
My preference here would be to keep the XML files as they were. The XML part of editorconfig was not discussed and likely was not applied before because we haven't edited them directly. Feel free to alter editorconfig to reach the previous formatting of XML files.

However, if our XML formatting is unconventional, then we could discuss how to fix it. But I suspect that it is not unconventional because it likley was generated automatically.

Reply:
Ah, the .editorconfig file already contains 2 spaces for .csproj files

![afbeelding](https://github.com/have-fun-was-taken/yafc-ce/assets/4461459/71a47ec5-65ff-4f1b-8783-611631feb21c)

Reply:
I ran some tests:

Visual studio scaffolding of .csproj files prefers 2 spaces independent of .editorconfig

Visual studio property editor prefers to respect existing space usage. So if the file has 4 spaces already for some xml nodes, new ones will be added at the same spacing.

Similar to RootNamespace. Visual studio prefers to keep it where it is, but will move it below TargetFramework if it can add it.

What i think is going on here is that these xml files are created by different versions of visual studio over the years, and the scaffolding might have contained different defaults before. It's also possible some manual reformatting has happened here and there.

The formatting itself doesn't harm anything, except readability in a file nobody is supposed to read. And it makes pull request change sets more annoying to review. So mostly my changes moved the files closer to what they would have been by default if created today. The main question remains if we want to take those changes with this Pr, or defer them to some other time?

Reply:
Thank you for the clarificaiton! The XML reformat can be done later.